### PR TITLE
Fixes to App.js

### PR DESCRIPTION
**Pull request by banciur**
Take a look how I changed App.js.

### CODE REVIEW

File: src/containers/App.js [hunk 1]
```diff
@@ -1,34 +1,46 @@
...
-    {props.rpcConnection()}
-    {props.web3 &&
-
-    <Grid fluid>
-      {getICOs().map(ico => <ICO key={ico.address} ico={ico} address={ico.address} />)}
-    </Grid>}
-    {!props.web3 &&
-    <div>
-      <h1>RPC connection Error</h1>
+/* Notice three changes:
```
Comment (krzkaczor) [inline src/containers/App.js]: please move this comment to PR description

File: package.json [hunk 2]
```diff
@@ -24,6 +24,7 @@
     "mocha": "^3.4.2",
     "moment": "^2.0.0",
     "node-sass": "^4.0.0",
+    "prop-types": "^15.5.10",
```
Comment (krzkaczor) [inline package.json]: rebase with master

**Comment by krzkaczor** [comment]
Uhh I just noticed that's really old PR. @banciur do you think that we still want to merge this? I mean maybe it was improved by other PRs. If not, then please close this PR.

**Comment by banciur** [comment]
It's old one partly outdated.

**Pull request closed without merging**